### PR TITLE
ADD: addition of subset_radar function in radar_utils

### PR DESCRIPTION
**Pull request by wolfidan**
Dear Py-ART team,

In this PR I propose the addition of a ~~cut_radar~~ subset_radar function from our MeteoSwiss fork of the library (thanks @jfigui). It cuts a radar object to a new domain specified by bounds in range, azimuth and elevation, and returns a new object. It can be useful to make processing faster when one is interested only in a certain subdomain around the radar (typically shorter ranges and only a certain sector).
This function is used a lot in our [pyrad](https://github.com/MeteoSwiss/pyrad) library, which is  a framework for real time radar processing that relies on PyART. 
I also added a short test function that checks if a cut radar object has indeed coordinates that match with the specified bounds.
Cheers,
Daniel W.

### CODE REVIEW

File: pyart/util/radar_utils.py [hunk 1]
```diff
@@ -104,6 +104,203 @@ def to_vpt(radar, single_scan=True):
     return
 
 
+def cut_radar(radar, field_names, rng_min=None, rng_max=None, ele_min=None,
```
Comment (mgrover1) [inline pyart/util/radar_utils.py:107]: This all looks great! One small suggestion - what would you think about changing the name here to subset_radar?

**Review by mgrover1 — CHANGES_REQUESTED** [review]
One small suggestion - other than that, looks great! Thanks for adding tests!

**Comment by wolfidan** [comment]
Thanks for the feedback @mgrover1, it's done

**Comment by wolfidan** [comment]
Could you help me with this failing check, I have honestly no idea what failed here in the website deploy?

**Comment by mgrover1** [comment]
@wolfidan we updated the trmm_rsl feedstock, changing the path in our CI. I added the fix here, should fix the problem. Thanks for making those changes!

**Review by mgrover1 — APPROVED** [review]

**Comment by zssherman** [comment]
@mgrover1 looks like it's still not finding trmm_rsl, the full path on condaforge has a weird extension now before the 4 and after the package name

**Comment by zssherman** [comment]
And it's different between osx and Linux for some reason... hmmm

**Comment by mgrover1** [comment]
I updated the TRMM rsl path - it was incorrect...

**Comment by wolfidan** [comment]
Thanks for your help!

**Comment by mgrover1** [comment]
@zssherman I think we can go ahead and merge - let's fix the TRMM_RSL for linux in a different PR

**Pull request merged**